pages/[user]/[type].tsx

```diff
@@ -1,22 +1,23 @@
-import { useEffect, useState } from "react";
-import { GetServerSideProps, GetServerSidePropsContext } from "next";
-import Head from "next/head";
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import { Availability } from ".prisma/client";
```

Comment:
@prisma/client has a benefit over .prisma/client due to always being present, whereas .prisma/client could still be generating.

```javascript
import { Availability } from "@prisma/client";
```

Reply:
👍  Just added what VSCode's autocompletion told me - not sure which is the recommended approach

```suggestion
import { Availability } from "@prisma/client";
```